src/card.css

```diff
@@ -0,0 +1,97 @@
+:root {
+  --color-aqua: #00acdc;
+  --color-white: #fff;
+  --color-grey: #616070;
+}
```

Comment:
That's good! A better approach is to keep it in a separate file or style.css, so please move it